### PR TITLE
[10x] trivial fix in the filling of fit charge MPV vs sensor thickness

### PR DESCRIPTION
**Pull request by mmusich**
Title says it all, the histograms are supposed to be 1D. 
Attention @dimattia

### CODE REVIEW

**Comment by ghellwig** [comment]
please test

**Comment by ghellwig** [comment]
+1
provided jenkins agrees

File: CalibTracker/SiStripChannelGain/src/SiStripGainsPCLHarvester.cc [hunk 1]
```diff
@@ -271,8 +271,8 @@ SiStripGainsPCLHarvester::gainQualityMonitor(DQMStore::IBooker& ibooker_, const
        if(FitMPV>0.) Gains->Fill(Gain);
  
        MPVs->Fill(FitMPV);
-       if(Thickness<0.04) MPVs320->Fill(Phi,FitMPV);
-       if(Thickness>0.04) MPVs500->Fill(Phi,FitMPV);
+       if(Thickness<0.04) MPVs320->Fill(FitMPV);
```
Comment (davidlange6) [inline CalibTracker/SiStripChannelGain/src/SiStripGainsPCLHarvester.cc]: hi @mmusich - does the ~same code with perhaps the same bug not need to be fixed in 
CalibTracker/SiStripChannelGain/plugins/SiStripGainFromCalibTree.cc

Comment (mmusich) [reply CalibTracker/SiStripChannelGain/src/SiStripGainsPCLHarvester.cc]: hello @davidlange6, right - thanks for the reminder. Done in the new commit.

**Comment by ghellwig** [comment]
please test

**Comment by davidlange6** [comment]
merge

**Pull request merged**